qlib/workflow/task/manage.py

```diff
@@ -80,6 +88,7 @@ def __init__(self, task_pool: str):
         task_pool: str
             the name of Collection in MongoDB
         """
+        #self.task_pool is mongodb's connection
         self.task_pool = getattr(get_mongodb(), task_pool)
```

Comment:
Why not using Python's annotation mechanism?
https://towardsdatascience.com/type-annotations-in-python-d90990b172dc